Mathlib/Analysis/Normed/Operator/LinearIsometry.lean

```diff
@@ -836,18 +843,17 @@ theorem mul_refl (e : E ≃ₗᵢ[R] E) : e * refl _ _ = e :=
 
 /-- Reinterpret a `LinearIsometryEquiv` as a `ContinuousLinearEquiv`. -/
 instance instCoeTCContinuousLinearEquiv : CoeTC (E ≃ₛₗᵢ[σ₁₂] E₂) (E ≃SL[σ₁₂] E₂) :=
-  ⟨fun e => ⟨e.toLinearEquiv, e.continuous, e.toIsometryEquiv.symm.continuous⟩⟩
+  ⟨fun e => e.toContinuousLinearEquiv⟩
```

Comment:
Is it true that there is no coercion to `ContinuousLinearEquiv` via the `EquivLike` hierarchy? If that doesn't exist, that's wonderful.